### PR TITLE
Reorg for converters tanh (FX Converter Refactor [4/N]) <Target: converter_reorg_proto>

### PR DESCRIPTION
**Pull request by apbose**

### CODE REVIEW

**Comment by narendasan** [comment]
@apbose please run linting.

File: py/setup.py [hunk 1]
```diff
@@ -145,6 +145,7 @@ def build_libtorchtrt_pre_cxx11_abi(develop=True, use_dist_dir=True, cxx11_abi=F
         cmd.append("--platforms=//toolchains:ci_rhel_x86_64_linux")
         print("CI based build")
 
+    cmd.append("--jobs=4")
```
Comment (narendasan) [inline py/setup.py:148]: Why was this change added?

**Review by wushirong — APPROVED** [review]
